app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -107,6 +107,51 @@ def pvm_instances
         :value        => instance.proc_type,
         :read_only    => true
       )
+
+      # saving pin_policy
+      persister.vms_and_templates_advanced_settings.build(
+        :resource     => ps_vmi,
+        :name         => 'pin_policy',
+        :display_name => _('Pin Policy'),
+        :description  => _('VM Pin Policy'),
+        :value        => instance.pin_policy,
+        :read_only    => true
+      )
+
+      next unless instance.software_licenses.nil?
+
+      ldesc = ""
+      if instance.software_licenses.ibmi_css
+        ldesc = "IBMi Cloud Storage Solution (ibmiCSS), "
+      end
+
+      if instance.software_licenses.ibmi_pha
+        ldesc += "IBMi Power High Availability (ibmiPHA), "
+      end
+
+      if instance.software_licenses.ibmi_rds
+        ldesc += "IBMi Rational Dev Studio(ibmiRDS)"
+        if instance.software_licenses.ibmi_rds_users
+          ldesc += " - (%d User Licenses)" % [instance.software_licenses.ibmi_rds_users]
+        end
+        ldesc += "#{ldec}, "
```

Comment:
I think this a straight bug because `ldec` doesn't exist.  But also I don't think you need this...

```suggestion
        ldesc += ", "
```